src/olympia/addons/migrations/0010_auto_20200604_1319.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 2.2.12 on 2020-06-04 13:19
+
+from django.db import migrations
+
+
+def delete_old_categories(apps, schema_editor):
+    AddonCategory = apps.get_model('addons', 'AddonCategory')
+    AddonCategory.objects.filter(
+        pk__in=(
+            # persona
```

Comment:
can we filter this qs based on `type` instead of hard-coding the ids?  There may be other non-constant defined Category instances still in the databases.

Reply:
Ha, right. I realized that I wasn't using the right model either, although it worked.. I changed my strategy anyway and removed all categories that we aren't supporting anymore.